src/silicone/multiple_infillers/infill_composite_values.py

```diff
@@ -22,14 +22,17 @@ def infill_composite_values(df, composite_dic=None):
     """
     if not composite_dic:
         composite_dic = {
-            "Emissions|PFC": ["Emissions|PFC|*"],
-            "Emissions|HFC": ["Emissions|HFC|*"],
+            "Emissions|PFC": [
```

Comment:
do we need to worry about other PFCs (e.g. C3F8, C4F10 etc.)? I'd be tempted to add them just in case but maybe they shouldn't be because there's something I don't understand about IAMs and reporting @jkikstra ?

Reply:
I've never seen data for any of them, I suppose it's free to include them though

Reply:
@znicholls those PFC gases are not on the latest variable definition/split up for the submission of scenarios that I have seen, and hence IAM teams should not report them - at least not for AR6. You could still add them if you'd like though, if that doesn't hurt. Maybe @byers has other or newer information, but I would say that there is no need to worry about them.

Reply:
cool won't worry about it then :)

Reply:
if we add all the lower ones, we should add C7F16 and C8F18 too